FusionIIIT/applications/globals/models.py

```diff
@@ -154,6 +161,11 @@ class ExtraInfo(models.Model):
         null=True, blank=True, upload_to='globals/profile_pictures')
     about_me = models.TextField(default='NA', max_length=1000, blank=True)
     date_modified = models.DateTimeField('date_updated', blank=True, null=True)
+    u_type = models.CharField(max_length=20, choices=Constants.USER_TYPES)
+    rem_medical_leave = models.IntegerField(default=15)
```

Comment:
Why are remaining leaves added in extrainfo? You can create another model in the ```leave``` application and link it with extrainfo as foreign key.

Reply:
Done, handled in LeaveCount instead.